Homework10/GenericArray.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Homework10
+{
+    public class GenericArray<T> where T : Human
+    {
+        private T[] _array;
+
+        public GenericArray(int size)
+        {
+            _array = new T[size + 1];
+        }
+
+        public int GetAraryLength() => _array.Length;
```

Comment:
If method is used only in this class make it private and move after all public and protected methods

Reply:
make sense, updated